src/ASM/ASMbase.h

```diff
@@ -312,6 +312,9 @@ class ASMbase
   virtual bool getParameterDomain(Real2DMat& u,
                                   IntVec* corners = nullptr) const = 0;
 
+  using NeighArray = std::vector<std::vector<int>>; //!< Convenience alias
```

Comment:
`IntMat` has been typedef'ed to the same thing in the top of the file. So you might consider
`using NeighArray = IntMat;`
if you want to use this name instead?

Reply:
I have changed to use IntMat.